### ISSUE TITLE
Know if a page is prerendered

### ISSUE THREAD

**Issue by florian-lefebvre** [BryceRussell/astro-pages#8]
Would be great to update the format of `pages` to be an `InjectedRoute`, including `prerender`. I guess it requires reading each file and some regex work

**Comment by BryceRussell** [comment]
Good idea!  I haven't explored adding `prerender` to injected pages yet but I know Arsh has, it seems like this would be a fairly easy to implement with Arsh's method of using a Vite plugin: https://github.com/lilnasy/gratelets/blob/main/packages/prerender-patterns/integration.ts#L28

(linking this for future [`InjectRoute`](https://github.com/withastro/astro/blob/3757a212c01cee03b8e96e1e9697a187ae7cc290/packages/astro/src/%40types/astro.ts#L1785))

**Comment by florian-lefebvre** [comment]
In the meantime I've made this little naive function
```ts
const isPrerendered = (str: string) => {
    const match = str.match(/export const prerender = (\w+)/)
    if (match) {
        return match[1] === "true"
    }
    return false;
}
```
